libmachine/provision/suse.go

```diff
@@ -135,7 +123,7 @@ func (provisioner *SUSEProvisioner) Provision(swarmOptions swarm.Options, authOp
 		return err
 	}
 
-	if strings.ToLower(provisioner.OsReleaseInfo.ID) != "opensuse" {
+	if !strings.HasPrefix(strings.ToLower(provisioner.OsReleaseInfo.ID), "opensuse") {
 		// This is a SLE machine, enable the containers module to have access
 		// to the docker packages
 		if _, err := provisioner.SSHCommand("sudo -E SUSEConnect -p sle-module-containers/12/$(uname -m) -r ''"); err != nil {
```

Comment:
Unrelated with the main topic of this PR, but here we hard code the `12` path which is no longer valid with SLE15

Reply:
Unfortunately, I haven't SLES and cannot check it, so I've just kept old code here. We should ask SLES expert, do we need this entire command and can we make it more flexible to support various SLES versions.

Reply:
@flavio Is `VERSION` field of `/etc/os-release` equals `15` for SLE 15? Can we just use `provisioner.OsReleaseInfo.Version` instead of hard-coded `12`?

Reply:
Leave it aside from this PR. I'll take care of that with another PR